lib/httpoison/base.ex

```diff
@@ -952,7 +952,16 @@ defmodule HTTPoison.Base do
        headers: process_response_headers.(headers),
        request: request,
        request_url: request.url,
-       redirect_url: :proplists.get_value("Location", headers, nil)
+       redirect_url: get_header(headers, "Location", nil)
```

Comment:
I wonder if we could use [`:hackney_headers.get_value`](https://github.com/benoitc/hackney/blob/master/doc/hackney_headers.md#get_value3) given that we already have hackney as a dependency?

Reply:
I had a look and gave it a try but it fails because that function (and `:hackney_headers_new.get_value` as well) expects the headers to be an Erlang [dict](https://www.erlang.org/doc/man/dict.html) while here it's a proplist (still haven't investigated why).

Reply:
Oh right! I didn't notice that! As you were then 😬